lib/subnet-routes.js

```diff
@@ -0,0 +1,79 @@
+/** @import { RequestWithSubnet } from './typings.js' */
+/** @import { FastifyInstance, FastifyReply } from 'fastify' */
+import { subnetPreHandlerHook } from './request-helpers.js'
+
+/**
+ * Define the subnet routes
+ * @param {FastifyInstance} app
+ */
+export const subnetRoutes = (app) => {
+  app.addHook('preHandler', subnetPreHandlerHook)
+
+  app.post(
+    '/:subnet/measurement',
+    {
+      schema: {
+        body: {
+          type: 'boolean'
+        }
+      }
```

Comment:
"nake" booleans are usually problematic, because it's not clear what the true/false value refers to. Also, if the request body is not an object but just a boolean value, then it's not open to extension - adding more measurement fields becomes a breaking change.

I know we want to keep this as simple as possible, and we don't have any immediate needs for adding more fields. ([YAGNI](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it)).

However, considering how small the change is, and how much effort it will save us in case we need to send more than one measurement field, I feel it's worth using an extensible schema from the start.

```suggestion
      schema: {
        body: {
          type: 'object',
          properties: {
            retrievalSucceeded: {
              type: 'boolean'
            }
          },
          required: ['retrievalSucceeded']
        }
      }
```